app/demo-loop/LDemoKernel.hh

```diff
@@ -18,6 +18,8 @@ using celeritas::StateHostRef;
 namespace demo_loop
 {
 //---------------------------------------------------------------------------//
+// TODO: convert all these to functors and autogenerate kernels
```

Comment:
Were you thinking we'd do the same with the track initialization kernels?

Reply:
Well, the stuff that has to use `thrust` algorithms are more up in the air. But in most of our loop demo kernels, the C++/CUDA kernel setup code are almost identical.